libs/utils/src/sentry_init.rs

```diff
@@ -0,0 +1,15 @@
+use sentry::ClientInitGuard;
+
+pub fn init_sentry(sentry_url: &str, process_name: &str) -> ClientInitGuard {
+    let _guard = sentry::init((
```

Comment:
Nit: This has been brought up already, but `_guard` is used here and thus should be renamed to `guard` instead.